src/ansys/dpf/composites/layup_info/_layup_info.py

```diff
@@ -371,6 +372,14 @@ def get_indexer_no_data_pointer(field: PropertyField) -> PropertyFieldIndexerSin
 
         self.mesh = mesh
         self.corner_nodes_by_element_type = _get_corner_nodes_by_element_type_array()
+        self.apdl_material_field = self.mesh.elements.materials_field
+
+        self.solver_material_to_dpf_id = {}
+        if solver_material_ids is not None:
```

Comment:
Enhance the ElementInfo by extracting the mapdl material ID from the RST and convert it into the DPF material ID.